compiler/driver.ml

```diff
@@ -309,9 +309,24 @@ let driver source_file (options : Cli.options) : int =
             in
             let prgm =
               if options.closure_conversion then (
+                if not options.avoid_exceptions then
+                  Errors.raise_error
+                    "Option --avoid_exceptions must be enabled for \
+                     --closure_conversion";
```

Comment:
Might be worth having `--avoid_exceptions` automatically be turned on in this case then ?

Reply:
Since `--avoid_exceptions` and `--closure_conversion` are experimental these days I want the CLI to force users to actively activate both and not have them activated silently.